### ISSUE TITLE
Plain text emails from dynamic templates

### ISSUE THREAD

**Issue by henadzit** [sklarsa/django-sendgrid-v5#62]
Hi,

It might be not an issue of this library, however, I would like to post this information in case someone else would run into similar issues. Today we noticed that our HTML dynamic template emails were sent out as plain text. We correlated the issue with [the recent change on the Sendgrid side](https://sendgrid.com/docs/ui/account-and-settings/retired-mail-settings/).

Out fix was to set content type explicitly

```
email = EmailMessage(from_email=email_from, to=[to])
# <<<
email.content_subtype = 'html'
# >>>

email.dynamic_template_data = ...
email.send(fail_silently=False)
```

Thanks,
Hena

**Comment by ivycheung7** [comment]
Having the same issue as well, sending it through the Email API works fine but when sending it through our platform which uses this library also sends it through plain text

**Comment by sklarsa** [comment]
I’ll take a look at this over the weekend and maybe I can come up with a fix that makes sense

**Comment by sklarsa** [comment]
Which version of sendgrid-python do you have installed?

**Comment by ivycheung7** [comment]
@sklarsa 0.8.1

**Comment by sklarsa** [comment]
What about your version of Sendgrid?  It should be something like v5 or v6

**Comment by ivycheung7** [comment]
Yep, Sendgrid v6

**Comment by sklarsa** [comment]
Can sendgrid dynamic templates ever have text content-types instead of html?  If so, then I don't think I can fix anything because you need additional api permissions to introspect your organization's sendgrid templates (if that's even possible).  However, if it is guaranteed that a dynamic template's content will always be html, then I can add some code to set the `content_subtype = 'html'` after (this line)[https://github.com/sklarsa/django-sendgrid-v5/blob/e1b46882d57f6eea8be4f176f2fb1434d5648512/sendgrid_backend/mail.py#L221]

**Comment by sklarsa** [comment]
closed by #68
